pootle/static/js/browser/components/TopContributorsTable.js

```diff
@@ -65,7 +65,7 @@ const TopContributorsTable = React.createClass({
       );
     }
     return (
-      <div>
+      <div className="bd">
```

Comment:
Note this breaks encapsulation; it's better to apply context-specific styles from parents (see 50320de).